### PR TITLE
[5.7] Moved the soft delete column name to the arg to make it consistent

### PR DESCRIPTION
**Pull request by SamAsEnd**
To make it consistent with [this](https://github.com/laravel/framework/blob/master/src/Illuminate/Database/Schema/Blueprint.php#L967)

screenshot from 2018-03-27 22-17-35

And to leverage [this](https://github.com/laravel/framework/blob/master/src/Illuminate/Database/Eloquent/SoftDeletes.php#L159)

screenshot from 2018-03-27 22-18-11

Thanks in advance.

### CODE REVIEW

**Comment by SamAsEnd** [comment]
To make it consistent with [this](https://github.com/laravel/framework/blob/master/src/Illuminate/Database/Schema/Blueprint.php#L967)

screenshot from 2018-03-27 22-17-35

And to leverage [this](https://github.com/laravel/framework/blob/master/src/Illuminate/Database/Eloquent/SoftDeletes.php#L159)

screenshot from 2018-03-27 22-18-11

Thanks in advance.

File: src/Illuminate/Database/Schema/Blueprint.php [hunk 1]
```diff
@@ -972,12 +973,13 @@ public function softDeletes($column = 'deleted_at', $precision = 0)
     /**
      * Add a "deleted at" timestampTz for the table.
      *
+     * @param  string  $column
      * @param  int  $precision
      * @return \Illuminate\Support\Fluent
      */
-    public function softDeletesTz($precision = 0)
+    public function softDeletesTz($column = 'deleted_at', $precision = 0)
```
Comment (SamAsEnd) [inline src/Illuminate/Database/Schema/Blueprint.php]: I think this one breaks backward compatibility, but not sure.

I will rebase it if you say so

Comment (TBlindaruk) [reply src/Illuminate/Database/Schema/Blueprint.php]: Please, move $column = 'deleted_at' to the second argument

File: src/Illuminate/Database/Schema/Blueprint.php [hunk 2]
```diff
@@ -380,21 +380,22 @@ public function dropTimestampsTz()
...
+        $this->dropColumn($column);
     }
 
     /**
      * Indicate that the soft delete column should be dropped.
      *
      * @return void
      */
-    public function dropSoftDeletesTz()
+    public function dropSoftDeletesTz($column = 'deleted_at')
```
Comment (browner12) [inline src/Illuminate/Database/Schema/Blueprint.php]: please add your docblock to this one.

File: src/Illuminate/Database/Schema/Blueprint.php (hunk 1 above)
Comment (browner12) [reply src/Illuminate/Database/Schema/Blueprint.php]: yes, it does. so you'll have to target 'master', which it looks like your are

File: src/Illuminate/Database/Schema/Blueprint.php (hunk 2 above)
Comment (SamAsEnd) [reply src/Illuminate/Database/Schema/Blueprint.php]: oops

**Comment by SamAsEnd** [comment]
Damn, i literally added a doc comment and travis fail and the funny part is the test it say is failed is `ThrottleRequestsWithRedisTest`

Can someone rerun this ...?

**Comment by SamAsEnd** [comment]
screenshot from 2018-03-28 06-56-28

**Comment by browner12** [comment]
Do we have a place to mark backwards compatible breaks? That might make it easier to write comprehensive upgrade guides for the next version being released

**Pull request merged**